include/pdal/Utils.hpp

```diff
@@ -149,17 +149,17 @@ namespace Utils
 
     inline std::string tolower(const std::string& s)
```

Comment:
Curious what the problem was the the old implementation.  Seems like it's doing exactly the same thing.  And what's with the 'X' bit?